models/linkages/README.md

```diff
@@ -0,0 +1,36 @@
+A generic template for adding a new model to PEcAn
```

Comment:
This README is specific to the template, and should be updated with something more informative about the LINKAGES model - in particular, a brief description, links, and references.